adafruit_sgp30.py

```diff
@@ -185,7 +193,7 @@ def set_iaq_relative_humidity(self, celsius, relative_humidity):
 
     # Low level command functions
 
-    def _run_profile(self, profile):
+    def _run_profile(self, profile: Tuple[str, List[int], int, float]) -> List[int]:
```

Comment:
I didn't review everything but thought it would be helpful to mention this first - technically this library uses lists for `profile` instead of `tuple` so it's not correct.  However, the way that this function unpacks the argument, I think tuples are a much better fit, and we should change all uses of this function to use tuples instead of lists.  Is that something you would be interesting in help with?  It'll make the type annotations much smoother and sensible.

Reply:
well, based on a comment to another pr, wouldn't we fix this to be lists for annotations and do the change to tuple in another pr?  I think I used tuple here because of hints pycharm was giving me and also the fact I was having problems wrapping my head around the structure of the profile being a list.  which says more about me and my understanding of lists, I suppose.  so yes, I should dig a bit and do that. not before next week, tho.  In here or a different PR?